sos/report/plugins/ceph_osd.py

```diff
@@ -28,13 +28,10 @@ def setup(self):
 
         # Only collect OSD specific files
         self.add_copy_spec([
+            "/run/ceph/ceph-osd*",
+            "/var/lib/ceph/osd/*/kv_backend",
             "/var/log/ceph/ceph-osd*.log",
             "/var/log/ceph/ceph-volume*.log",
-
-            "/var/lib/ceph/osd/",
-            "/var/lib/ceph/bootstrap-osd/",
```

Comment:
Is this really desired to stop collecting?

Reply:
This doesn't contain anything useful as it's used during the osd bootstrapping. Typically contains just the keyring (which is excluded already deliberately).

Reply:
OK then :+1: . I wasn't sure if it is deliberate or a mistake.